world_universities_and_domains.json

```diff
@@ -127240,5 +127230,414 @@
+    "alpha_two_code": "MX",
+    "state-province": null,
+    "domains": [
+      "utsc.edu.mx"
+    ],
+    "country": "Mexico"
+  },
+  {
+    "web_pages": [
+      "https://unidad.gdl.cinvestav.mx/"
+    ],
+    "name": "CINVESTAV",
+    "alpha_two_code": "MX",
+    "state-province": null,
+    "domains": [
+      "cinvestav.mx"
+    ],
+    "country": "Mexico"
+  },
+  {
+    "web_pages": [
+      "https://msa.edu.eg/"
+    ],
+    "name": "MSA University",
+    "alpha_two_code": "EG",
+    "state-province": null,
+    "domains": [
+      "msa.edu.eg"
+    ],
+    "country": "Egypt"
+  },
+  {
+    "web_pages": [
+      "https://www.pm.szczecin.pl/en/"
+    ],
+    "name": "Maritime University of Szczecin",
+    "alpha_two_code": "PL",
+    "state-province": null,
+    "domains": [
+      "pm.szczecin.pl"
+    ],
+    "country": "Poland"
+  },
+  {
+    "web_pages": [
+      "https://taltech.ee/en/"
+    ],
+    "name": "Tallinn University of Technology",
+    "alpha_two_code": "EE",
+    "state-province": null,
+    "domains": [
+      "taltech.ee"
+    ],
+    "country": "Estonia"
+  },
+  {
+    "web_pages": [
+      "https://korea.ac.kr/"
+    ],
+    "name": "Korea University",
+    "alpha_two_code": "KR",
+    "state-province": null,
+    "domains": [
+      "korea.ac.kr"
+    ],
+    "country": "South Korea"
+  },
+  {
+    "web_pages": [
+      "https://www.gu.edu.eg/"
+    ],
+    "name": "Galala University",
+    "alpha_two_code": "EG",
+    "state-province": null,
+    "domains": [
+      "gu.edu.eg"
+    ],
+    "country": "Egypt"
+  },
+  {
+    "web_pages": [
+      "https://www.usk.ac.id"
+    ],
+    "name": "Universitas Syiah Kuala",
+    "alpha_two_code": "ID",
+    "state-province": null,
+    "domains": [
+      "che.usk.ac.id"
+    ],
+    "country": "Indonesia"
+  },
+  {
+    "web_pages": [
+      "https://www.ucv.ro/"
+    ],
+    "name": "University of Craiova",
+    "alpha_two_code": "RO",
+    "state-province": null,
+    "domains": [
+      "student.ucv.ro"
+    ],
+    "country": "Romania"
+  },
+  {
+    "web_pages": [
+      "https://cu.edu.eg/index2.php"
+    ],
+    "name": "Cairo University",
+    "alpha_two_code": "EG",
+    "state-province": null,
+    "domains": [
+      "stud.fci-cu.edu.eg"
+    ],
+    "country": "Egypt"
+  },
+  {
+    "web_pages": [
+      "https://www.ifgoiano.edu.br"
+    ],
+    "name": "Instituto Federal Goiano Campus Morrinhos",
+    "alpha_two_code": "BR",
+    "state-province": null,
+    "domains": [
+      "ifgoiano.edu.br"
+    ],
+    "country": "Brazil"
+  },
+  {
+    "web_pages": [
+      "https://cspc.edu.ph/"
+    ],
+    "name": "Camarines sur polytechnic colleges",
+    "alpha_two_code": "PH",
+    "state-province": null,
+    "domains": [
+      "my.cspc.edu.ph"
+    ],
+    "country": "Philippines"
+  },
+  {
+    "web_pages": [
+      "https://www.ump.edu.my/en"
+    ],
+    "name": "UNIVERSITI MALAYSIA PAHANG AL- SULTAN ABDULLAH",
```

Comment:
Probably shouldn't be all caps

Reply:
Corrected at b61d4dfa7aec53e70072ef1feb36ce66f44bdcc8